src/Components/WeatherCard/index.tsx

```diff
@@ -0,0 +1,67 @@
+import { useContext } from "react";
+import { WeatherStateData } from "../../Context";
+import cloud from "../../assest/icons/cloud.png";
+
+import { useHistory } from "react-router-dom";
+
+// const WeatherCard = () => {
+//   const context = useContext(WeatherStateData);
+//   const history = useHistory();
+//   return (
+//     <div
+//       className="weather-div"
+//       onClick={() => history.push(`/${context.city}/${context.day}`)}
+//     >
+//       <p className="card-heading">Friday</p>
+//       <p className="card-text">March 1st, 1:00pm</p>
+//       <img className="image" src={cloud} alt="cloud" />
+//       <h1>35 &#8457;</h1>
+//       <p>Clear Sky</p>
+//     </div>
+//   );
+// };
+// export default WeatherCard;
+
+import * as React from "react";
+import Card from "@mui/material/Card";
+import CardContent from "@mui/material/CardContent";
+import Typography from "@mui/material/Typography";
+import Avatar from "@mui/material/Avatar";
```

Comment:
The imports always goes to top of the file

Reply:
ok